.circleci/config.yml

```diff
@@ -101,12 +105,29 @@ commands:
     description: Restore Cargo package cache from prev job
     steps:
       - restore_cache:
+          name: Restore cargo package cache
           key: cargo-package-cache-{{ checksum "Cargo.lock" }}
       - run:
           name: Check cargo package cache
           command: |
             ls -all /usr/local/cargo
             du -ssh /usr/local/cargo
+  save_breaking_change_rev:
+    description: Save the breaking change rev since last testnet update.
+    steps:
+      - save_cache:
+          name: Save breaking change rev
+          key: testnet-{{ checksum "testnet_rev" }}
```

Comment:
Do you have to hardcode the paths even though it's in an ENV var in the `check-breaking-change` step?

Reply:
Unfortunately I have to hard coded. tldr - the save_cache/restore_cache cmds don't take random ENV VARs we defined. The cache key is literally the `testnet-${TESTNET_REV}` string.  There is an explanation note in lines 458 and 468 below.